tests/unit/support/queue.ts

```diff
@@ -166,5 +167,51 @@ registerSuite({
 		setTimeout(dfd.callback(function () {
 			assert.equal(parts.join(','), 'start,end');
 		}), 100);
+	},
+
+	'web workers': {
+		'queue from webworker': function (this: any) {
+			if (global.Worker === undefined) {
+				this.skip('does not support web workers');
+				return;
+			}
+
+			const baseUrl = location.origin;
+			const dfd = this.async(10000);
+			const blob = new Blob([ `(function() { 
```

Comment:
I ended up adding a simple `global.Blob === undefined` instead of a *real* Blob has check. The real Blob check is in `core` and has a few dependencies that we'd also have to copy over. Maybe that would be worth while at some point?